### PR TITLE
Fix livesearch in discopower

### PR DESCRIPTION
**Pull request by nikosev**
This PR contains the following changes:
- Replace data['header'] with data['htmlinject']['htmlContentPost'] to load javascript scripts to templates
- Fix undefined index error for `idps_in_searchable_list_index`
- Add paths for jQuery
- Add login_with key in discopower definitions
- Move ribbon_text to config file

### CODE REVIEW

**Comment by nikosev** [comment]
@olgabrani Could you please review this PR?

**Comment by nikosev** [comment]
[Update vanilla theme for SSP v1.17+](https://trello.com/c/rda7ZDTt/181-update-vanilla-theme-for-ssp-v117)

File: themes/ssp/discopower/disco.tpl.php [hunk 1]
```diff
@@ -25,20 +25,26 @@
...
 
-
-$this->data['head'] = '<script type="text/javascript" src="'.
-    SimpleSAML\Module::getModuleURL('discopower/assets/js/jquery.livesearch.js').'"></script>'."\n";
-$this->data['head'] .= '<script type="text/javascript" src="'.
-    SimpleSAML\Module::getModuleURL('discopower/assets/js/' . $this->data['score'].'.js')  . '"></script>'."\n";
-
-$this->data['head'] .= $this->data['search'];
+$this->data['head'] = '<link rel="stylesheet" media="screen" type="text/css" href="'.
+    SimpleSAML\Module::getModuleURL('discopower/assets/css/uitheme1.12.1/jquery-ui.min.css').'" />';
```
Comment (olgabrani) [inline themes/ssp/discopower/disco.tpl.php:31]: @nikosev why did you have to load ui css theme?

Comment (nikosev) [reply themes/ssp/discopower/disco.tpl.php:31]: @olgabrani I added the CSS to be compatible with the latest version of discopower which supports SimpleSAMLphp v1.18
https://github.com/simplesamlphp/simplesamlphp-module-discopower/blob/release-0.9/templates/disco.tpl.php#L3-L25
Also this change is fixing the loading issue of the languages

**Comment by olgabrani** [comment]
LGTM

**Pull request merged**